conanfile.py

```diff
@@ -327,19 +350,20 @@ def add_basilisk_to_sys_path(self):
     # XXX: This needs to be run before dispatching to Conan (i.e. outside of the
     # ConanFile object), because it affects the configuration of the first run.
     # (Running it here fixes https://github.com/AVSLab/basilisk/issues/525)
-    if platform.system() != "Darwin":
+    try:
+        subprocess.check_output(["conan", "profile", "detect", "--exist-ok"])
+    except:
+        # if profile already exists the above command returns an error.  Just ignore in this
+        # case.  We don't want to overwrite an existing profile file
+        pass
+
+    if platform.system() == "Linux":
         try:
-            subprocess.check_output(["conan", "profile", "new", "default", "--detect"])
+            # XXX: This fixes a linker issue due to the dual C++ ABI.
+            subprocess.check_output(["conan", "profile", "update", "settings.compiler.libcxx=libstdc++11", "default"])
```

Comment:
@schaubh (This is only minor) Just FYI, this command does not work on Conan2, so this will always fail. There is no `conan profile update` anymore.

Reply:
Howdy @dpad , thanks for the feedback.  The Linux CI builds seem to run fine.  Are you recommending I remove this Linux specific conan command?  It is actually specifying c++11 which doesn't look right to me.

Reply:
If I'm not mistaken, with Conan 2, this line will always fail and go straight into the exception, which does nothing. So this line is doing nothing.

Reply:
I'll make a branch to remove this dead code.  Thanks for the feedback, especially on Linux systems.

Reply:
@schaubh The c++11 actually refers to the breaking change specific to gcc libstdc++ ABI implemented for c++11. https://gcc.gnu.org/onlinedocs/libstdc++/manual/using_dual_abi.html

FWIW, I just tried `conan profile detect` on a Linux VM (Ubuntu 24.04) and it automatically chooses `libstdc++11`, so even if the command worked, it would be redundant.